score/etl/score/etl_county_state_score.py

```diff
@@ -0,0 +1,89 @@
+import pandas as pd
+
+from etl.base import ExtractTransformLoad
+from utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class CountyStateScoreETL(ExtractTransformLoad):
+    def __init__(self):
+        self.CENSUS_COUNTIES_ZIP_URL = "https://www2.census.gov/geo/docs/maps-data/data/gazetteer/2020_Gazetteer/2020_Gaz_counties_national.zip"
```

Comment:
We'll want to use the 2010 version of this to match the census data we're currently pulling - that's linked to from here: https://www.census.gov/geographies/reference-files/time-series/geo/gazetteer-files.2010.html

Reply:
Updated